local/jimm/qa-lxd-mac-forward.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+# This script forwards JIMM and Keycloaks ports to localhost from the QA VM.
+#
+# It can be used in a number of ways, as follows:
+# ./qa-lxd-mac-forward.sh jimm2 --linux --wait  - Run for a VM named jimm2, on linux, and wait after tunnels are established
+# ./qa-lxd-mac-forward.sh jimm2 --linux         - Run for a VM named jimm2, on linux, and exit immediately
+# ./qa-lxd-mac-forward.sh --linux               - Run for the default VM (jimm), on linux, and exit immediately
+# ./qa-lxd-mac-forward.sh jimm2 --mac --wait    - Run for a VM named jimm2, on mac, and wait after tunnels are established
+
+KEY_PATH_LINUX="/var/snap/multipass/common/data/multipassd/ssh-keys/id_rsa"
+KEY_PATH_MAC="/var/root/Library/Application Support/multipassd/ssh-keys/id_rsa"
+VM_NAME="${VM_NAME:-jimm}"
+OS=""
+WAIT=false
+
+while [[ $# -gt 0 ]]; do
+  case "$1" in
+    --linux)
+      OS="linux"
+      shift
+      ;;
+    --mac)
+      OS="mac"
+      shift
+      ;;
+    --wait)
+      WAIT=true
+      shift
+      ;;
+    *)
+      # Assume positional argument VM_NAME or unknown
+      VM_NAME="$1"
+      shift
+      ;;
+  esac
+done
+
+# Set the key path based on OS
+if [[ "$OS" == "mac" ]]; then
+  KEY_PATH="$KEY_PATH_MAC"
+elif [[ "$OS" == "linux" ]]; then
+  KEY_PATH="$KEY_PATH_LINUX"
+else
+  echo "Error: You must specify --linux or --mac"
+  exit 1
+fi
+
+echo "Copying ssh key to local"
+KEY_COPIED_PATH="../../local/vm/id_rsa"
+if [ ! -f "$KEY" ]; then
```

Comment:
Should this be 
```suggestion
if [ ! -f "$KEY_COPIED_PATH" ]; then
```
? Or are you trying to allow the user to provide a key via an env var?

Reply:
Yes! Good spot